src/stories/HsitorySelect/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+import {Button, ButtonProps, HistorySelectionDialogTitle, SelectionModal} from '../../components';
+
+export interface HistorySelectionProps extends ButtonProps{}
+
+export default function HistorySelection() {
+    const [open, setOpen] = useState(false);

```

Comment:
better to use useReducer hook, since handleClose is thrown through <SelectionModal> but not used there , instead it's used in <SelectionTable>